Kickstarter-iOS/Views/Controllers/FeatureFlagToolsViewController.swift

```diff
@@ -0,0 +1,109 @@
+import Foundation
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class FeatureFlagToolsViewController: UITableViewController {
+  private let viewModel = FeatureFlagToolsViewModel()
+
+  // MARK: - Properties
+
+  private var features = [(Feature, Bool)]()
+  private let reuseId = "FeatureFlagTools.TableViewCell"
```

Comment:
You can register a cell in more type safe manner using its class name like we do elsewhere `self.tableView.registerCellClass(SettingsTextInputCell.self)` and then reuse it like so `tableView.dequeueReusableCell(withClass: SettingsTextInputCell.self, for: indexPath)` ... unless there was an issue with the base `UITableViewCell` class?

Reply:
Yeah just registering a basic `UITableViewCell` breaks `classNameWithoutModule`